File: workspace/data-model/src/main/java/de/pfiva/data/model/survey/Response.java
package de.pfiva.data.model.survey;

import java.io.Serializable;
import java.util.List;

import de.pfiva.data.model.User;

public class Response implements Serializable {

	private static final long serialVersionUID = 1L;
	
	private int id;
	private int questionId;
	private List<String> values;
	private User user;
	
	public int getId() {
		return id;
	}
	public void setId(int id) {
		this.id = id;
	}
	public int getQuestionId() {
		return questionId;
	}
	public void setQuestionId(int questionId) {
		this.questionId = questionId;
	}
	public List<String> getValues() {
		return values;
	}
	public void setValues(List<String> values) {
		this.values = values;
	}
	public User getUser() {
		return user;
	}
	public void setUser(User user) {
		this.user = user;
	}
	@Override
	public String toString() {
		return "Response [id=" + id + ", questionId=" + questionId + ", values=" + values + ", user=" + user + "]";
	}
}

File: angular_workspace/pfiva-web/src/app/data-model/survey/Response.ts
import { User } from "../User";

export class Response {
	private id: number;
	private questionId: number;
	private values: string[];
	private user: User;

	public get Id(): number {
		return this.id;
	}
	public set Id(value: number) {
		this.id = value;
	}
	public get QuestionId(): number {
		return this.questionId;
	}
	public set QuestionId(value: number) {
		this.questionId = value;
	}
	public get Values(): string[] {
		return this.values;
	}
	public set Values(value: string[]) {
		this.values = value;
	}
	public get User(): User {
		return this.user;
	}
	public set User(value: User) {
		this.user = value;
	}
}